spec/features/admin/business_units_spec.rb

```diff
@@ -39,7 +39,7 @@
 
     teams_areas_page.add_area_field.set 'This is another area'
     teams_areas_page.add_button.click
-    teams_areas_page.wait_until_existing_areas_visible count: 2
+    sleep(2)
```

Comment:
IMO not remove the line and move the sleep(2) above this line, the count of 2 means 2 elements not 2 seconds?

Reply:
Yes I think you're correct - though as it is not an expectation and it passes with the sleep I think it is okay.